config.sh

```diff
@@ -28,12 +28,85 @@ readonly HOST_LLVM_PATH="${HOST_LLVM_PATH:-/usr/bin}"
 # How many CPUs to use for building the LLVM baremetal toolchain for Arm:
 readonly NPROC="${NPROC:-$(getconf _NPROCESSORS_ONLN)}"
 
-# Default triple of the toolchain:
-readonly TARGET=armv6m-none-eabi
+
+LIBRARY_SPEC="${LIBRARY_SPEC:-armv6m}"
```

Comment:
This is expected to be the default, however it is not in the list below _soft_nofp suffix is missing.

Reply:
fixed